src/test/java/com/artipie/rpm/RepoConfigFromYamlTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm;
+
+import com.amihaiemil.eoyaml.Yaml;
+import java.util.Optional;
+import org.cactoos.func.ProcOf;
+import org.cactoos.list.ListOf;
+import org.hamcrest.Matcher;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.AllOf;
+import org.junit.jupiter.api.Test;
+import org.llorllale.cactoos.matchers.MatcherOf;
+
+/**
+ * Test for {@link RepoConfig.FromYaml}.
+ * @since 0.10
+ */
+public final class RepoConfigFromYamlTest {
+
+    @Test
+    void readsSettings() {
+        MatcherAssert.assertThat(
+            new RepoConfig.FromYaml(
+                Yaml.createYamlMappingBuilder().add("digest", "sha1")
+                .add("naming-policy", "sha256").add("filelists", "false").build()
+            ),
+            new AllOf<RepoConfig.FromYaml>(
```

Comment:
@olenagerasimova this can be simplified using `Matchers.allOf` to avoid wrapping to list